api/tests/Api/ContentNodes/ChecklistNode/UpdateChecklistNodeTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace App\Tests\Api\ContentNodes\ChecklistNode;
+
+use App\Entity\ContentNode\ChecklistNode;
+use App\Tests\Api\ContentNodes\UpdateContentNodeTestCase;
+
+/**
+ * @internal
+ */
+class UpdateChecklistNodeTest extends UpdateContentNodeTestCase {
```

Comment:
are there tests which use addChecklistItemIds or removeChecklistItemIds?

Reply:
Yes - in this file.
See line 37, 55, 74, 88, 99